pkg/controller/framework/controller_test.go

```diff
@@ -335,7 +335,12 @@ func TestUpdate(t *testing.T) {
 		},
 	)
 
+	const threads = 3
+	testDoneWG.Add(threads * len(tests))
```

Comment:
The issue with Shippable is that this test is failing to build, probably because `tests` isn't defined yet when it's used here.